BUILD

```diff
@@ -0,0 +1,247 @@
+        "@com_github_stretchr_testify//suite",
+    ],
+)
+
+go_test(
+    name = "unarchive_test",
+    srcs = ["unarchive_test.go"],
+    data = [
+        ":ansible-puller.json",
+        ":testdata",
+    ],
+    embed = [":ansible_puller_lib"],
+    deps = [
+        "@com_github_stretchr_testify//assert",
+        "@com_github_stretchr_testify//suite",
+    ],
+)
+
+#
+# deb
+#
+
+go_binary(
+    name = "ansible_puller_bin",
+    basename = "ansible-puller",
+    embed = [":ansible_puller_lib"],
+    visibility = ["//visibility:public"],
+)
+
+pkg_tar(
+    name = "pkg_tar_ansible_puller_bin",
+    srcs = [":ansible_puller_bin"],
+    mode = "0755",
+    package_dir = "/usr/bin",
+)
+
+pkg_tar(
+    name = "pkg_tar_ansible_puller_config",
+    srcs = [":ansible-puller.json"],
+    mode = "0755",
+    package_dir = "/etc/ansible-puller/",
+)
+
+pkg_tar(
+    name = "pkg_tar_ansible_puller_systemd",
+    srcs = [":ansible-puller.service"],
+    mode = "0755",
+    package_dir = "/etc/systemd/system/",
+)
+
+pkg_tar(
+    name = "pkg_tar_ansible_puller",
+    extension = "tar.gz",
+    deps = [
+        ":pkg_tar_ansible_puller_bin",
+        ":pkg_tar_ansible_puller_config",
+        ":pkg_tar_ansible_puller_systemd",
+    ],
+)
+
+#
+# rpm
+#
+
+VERSION = "1"
+
+RELEASE = "0"
+
+pkg_deb(
+    name = "ansible_puller_deb",
+    architecture = "amd64",
+    data = ":pkg_tar_ansible_puller",
+    description = "This daemon extends the ansible-pull method of running Ansible. It uses S3 or HTTP file transmission instead of Git to manage distribution (easy to cache), and integrates with Prometheus monitoring.",
+    homepage = "https://github.com/teslamotors/ansible_puller",
+    maintainer = "authors of ansible_puller",
+    package = "ansible-puller",
+    package_file_name = "ansible-puller.deb",
+    version = VERSION,
+)
+
+pkg_files(
+    name = "pkg_file_ansible_puller_json",
+    srcs = [":ansible-puller.json"],
+    prefix = "/etc/ansible-puller/",
+)
+
+pkg_files(
+    name = "pkg_file_ansible_puller_bin",
+    srcs = ["//:ansible_puller_bin"],
+    attributes = pkg_attributes(mode = "0755"),
+    prefix = "/usr/bin/",
+)
+
+pkg_files(
+    name = "pkg_file_ansible_puller_service",
+    srcs = [":ansible-puller.service"],
+    prefix = "/etc/systemd/system/",
+)
+
+pkg_mkdirs(
+    name = "pkg_mkdirs_ansible_puller_dirs",
+    dirs = [
+        "/var/log/ansible-puller/",
+    ],
+)
+
+pkg_filegroup(
+    name = "ansible_puller_rpm_files",
+    srcs = [
+        ":pkg_file_ansible_puller_bin",
+        ":pkg_file_ansible_puller_json",
+        ":pkg_file_ansible_puller_service",
+        ":pkg_mkdirs_ansible_puller_dirs",
+    ],
+)
+
+pkg_rpm(
+    name = "ansible_puller_rpm",
+    srcs = [":ansible_puller_rpm_files"],
+    description = "This daemon extends the ansible-pull method of running Ansible. It uses S3 or HTTP file transmission instead of Git to manage distribution (easy to cache), and integrates with Prometheus monitoring.",
```

Comment:
You could put this in a variable if you want so that it stays consistent between the two packages.